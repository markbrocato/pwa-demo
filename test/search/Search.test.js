import React, { Component } from 'react';
import { mount } from 'enzyme'
import Search from '../../src/search/Search'

describe('Search', () => {
  it('should render the search view', () => {
    expect(mount(<Search/>)).toMatchSnapshot()
  })
})