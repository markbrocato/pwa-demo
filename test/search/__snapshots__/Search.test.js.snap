// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search should render the search view 1`] = `
<Search>
  <div>
    Search
  </div>
</Search>
`;
